python/ribasim_testmodels/ribasim_testmodels/pid_control.py

```diff
@@ -77,7 +77,7 @@ def pid_control_model():
     state = pd.DataFrame(
         data={
             "node_id": [2],
-            "storage": [500.0],
+            "level": [0.5],
         }
     )
 
```

Comment:
Is the QGIS change missing? https://github.com/Deltares/Ribasim/blob/b1a695c1d7abaffa38b18a43d09823064bc577a5/qgis/core/nodes.py#L326

Reply:
Yep, thanks